onnx/defs/tensor/defs.cc

```diff
@@ -1395,40 +1395,94 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrain input 'X' and output 'Y' to all tensor types.")
         .SetDoc(Upsample_ver10_doc)
         .TypeAndShapeInferenceFunction([](InferenceContext& ctx) {
-          resizeShapeInference(ctx);
+          resizeShapeInference(ctx, false);
         }));
 
-static const char* Resize_ver10_doc = R"DOC(
+static const char* Resize_ver11_doc = R"DOC(
 Resize the input tensor.
 Each dimension value of the output tensor is:
-  output_dimension = floor(input_dimension * scale).
+  output_dimension = floor(input_dimension * scale) if input \"sizes\" is not specified.
 )DOC";
 
 ONNX_OPERATOR_SET_SCHEMA(
     Resize,
-    10,
+    11,
     OpSchema()
         .Attr(
             "mode",
-            "Two interpolation modes: nearest (default), and linear (including bilinear, trilinear, etc)",
+            "Three interpolation modes: nearest (default), linear and cubic. "
+            "The \"linear\" mode includes linear interpolation for 1D tensor and N-D linear interpolation for N-D tensor (For example, bilinear interpolation for 2D tensor). "
+            "The \"cubic\" mode includes cubic interpolation for 1D tensor and N-D cubic interpolation for N-D tensor (For example, bicubic interpolation for 2D tensor)."
```

Comment:
I can't figure out how to implement this operator after reading this spec. Are you using this equation (copied from your reference paper)?
![image](https://user-images.githubusercontent.com/3524474/62344992-3eef8980-b4a5-11e9-8ac7-fff5cb24ca52.png)

Reply:
N-D (2-D is a special case where N=2) interpolation is implemented by performing 1-D interpolation recursively. For N=2, assume we want to get the value at `(1.6, 2.8)` in resized 2-D HxW tensor, we firstly calculate H values at `(1.6)` by performing 1-D interpolation on H rows (a.k.a. H 1-D tensors) respectively, and then calculate the value at `(2.8)` of the generate line (a.k.a. 1-D tensor) consisting of H above values. Similarly, 3-D interpolation is 2-D interpolation on every layer (a.k.a. every 2-D tensor) and 1-D interpolation on the generated line (a.k.a. 1-D tensor).